hooks/pre/30-upgrade.rb

```diff
@@ -2,6 +2,22 @@ def migrate_candlepin
   Kafo::Helpers.execute("/usr/share/candlepin/cpdb --update --password #{Kafo::Helpers.read_cache_data('candlepin_db_password')}")
 end
 
+def migrate_gutterball
+  tomcat_dir = Kafo::Helpers.read_cache_data('gutterball_tomcat')
+  username = Kafo::Helpers.read_cache_data('gutterball_db_user')
+  password = Kafo::Helpers.read_cache_data('gutterball_db_password')
+  migration_command = <<-END_OF_COMMAND
+/usr/bin/liquibase --driver=org.postgresql.Driver \
+        --classpath=/usr/share/java/postgresql-jdbc.jar:/var/lib/#{tomcat_dir}/webapps/gutterball/WEB-INF/classes/ \
+        --changeLogFile=db/changelog/changelog.xml \
+        --url=jdbc:postgresql:gutterball \
+        --username=#{username}\
+        --password=#{password}\
+        update
+END_OF_COMMAND
```

Comment:
@dustints what do you think about turning this into a script? Since it is being used in 2 places (here and in the puppet module) if we deploy and manage it as a script then we reduce managing this in 2 places. Further, the script could read from gutterball.conf on the filesystem since we are already storing the username/password within and then we don't have to worry about reading the values or caching them.